ivadomed/models.py

```diff
@@ -1440,6 +1441,80 @@ def forward(self, x):
+        self.in_channel = in_channel
+        self.out_channel = out_channel
+        self.features = features
+        self.attention = attention
+        self.dropout_rate = dropout_rate
+        self.final_activation = final_activation
+
+        ivadomed_params = {**self.__dict__, **kwargs}
+        monai_model_keys = ['features', 'act', 'norm', 'bias', 'upsample']
+        monai_params = ivadomed_params_to_monai_params(ivadomed_params, monai_model_keys)
+
+        self.model = basic_unet.BasicUNet(**monai_params)
+
+    def forward(self, x):
+        out = self.model(x)
+
+        if hasattr(self, "final_activation") and self.final_activation not in ["softmax", "relu", "sigmoid"]:
+            raise ValueError("final_activation value has to be either softmax, relu, or sigmoid")
+        elif hasattr(self, "final_activation") and self.final_activation == "softmax":
+            out = self.softmax(out)
+        elif hasattr(self, "final_activation") and self.final_activation == "relu":
+            out = nn.ReLU()(out) / nn.ReLU()(out).max() if bool(nn.ReLU()(out).max()) else nn.ReLU()(out)
+        else:
+            out = torch.sigmoid(out)
+
+        return out
+
+
+class UNETR(Module):
+    """MONAI'S UNETR model (<https://arxiv.org/abs/2103.10504>) adopted in ivadomed"""
+
+    def __init__(self, in_channel, out_channel, feature_size=16, hidden_size=768, mlp_dim=3072,
+                 num_heads=12, pos_embed="conv", norm_name="instance", dropout_rate=0.3, 
+                 final_activation="sigmoid", **kwargs):
+        super(UNETR, self).__init__()
+        self.in_channel = in_channel
+        self.out_channel = out_channel
+        self.img_size = kwargs['length_3D']
+        self.feature_size = feature_size
+        self.hidden_size = hidden_size
+        self.mlp_dim = mlp_dim
+        self.num_heads = num_heads
+        self.pos_embed = pos_embed
+        self.norm_name = norm_name
+        self.dropout_rate = dropout_rate
+        self.final_activation = final_activation
+
+        ivadomed_params = {**self.__dict__, **kwargs}
+        monai_model_keys = ['img_size', 'feature_size', 'hidden_size', 'mlp_dim', 'num_heads', 'pos_embed', 'norm_name']
+        monai_params = ivadomed_params_to_monai_params(ivadomed_params, monai_model_keys)
+        self.model = unetr.UNETR(**monai_params)
+
+    def forward(self, x):
+        out = self.model(x)
+
+        if hasattr(self, "final_activation") and self.final_activation not in ["softmax", "relu",
+                                                                               "sigmoid"]:
+            raise ValueError("final_activation value has to be either softmax, relu, or sigmoid")
+        elif hasattr(self, "final_activation") and self.final_activation == "softmax":
+            out = self.softmax(out)
+        elif hasattr(self, "final_activation") and self.final_activation == "relu":
+            out = nn.ReLU()(out) / nn.ReLU()(out).max() if bool(
```

Comment:
In this case, using the function API would be best. (Remove next line if suggested code is accepted.)
```suggestion
            out = F.relu(out) / F.relu(out).max() if bool(
                F.relu(out).max()) else F.relu(out)
```